Cohort2/team4/client/src/components/LandingPage.tsx

```diff
@@ -0,0 +1,14 @@
+// Landing Page
```

Comment:
Are we using this component at all? Not sure we need this, and the index.tax file doesn't reference it.